dbinit.sql

```diff
@@ -4,130 +4,97 @@ SET sql_safe_updates = false;
 DROP DATABASE IF EXISTS movr CASCADE; 
 
 
-CREATE DATABASE movr;
+CREATE DATABASE movr PRIMARY REGION "gcp-us-east1" REGIONS "gcp-europe-west1", "gcp-us-west1";
```

Comment:
the problem with this setup is that users have to ensure they set up the servers correctly, i.e. set `--locality=region=<blah>` in their flask setup. is that what we want / already have?

Reply:
Yeah ... the README and tutorial specifically instruct the user to specify the localities that match these region names. They are based on the GCP cluster names. On CC, the localities for a multi-region GCP cluster should match these names anyways.

Reply:
TFTR @otan !